src/Gwen/APIHandlers/SpotifyAPI/load_cookies.py

```diff
@@ -0,0 +1,168 @@
+import pickle
+import spotipy
+import cred
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.chrome.options import Options
+from spotipy.oauth2 import SpotifyOAuth
+import time
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import TimeoutException
+
+class Spotify:
+
+
+    #chrome driver
+    webdriver_service = Service(".\chromedriver.exe")
+
+    #options
+    chrome_options=Options()
+    chrome_options.add_argument("--no-sandbox")
+    chrome_options.add_experimental_option("detach", True)
+    #option to remove error: Filed to read descriptor from node connection: A device attached to the system is not functioning
+    chrome_options.add_experimental_option('excludeSwitches', ['enable-logging'])
+
+    #create browser
+    browser = webdriver.Chrome(service=webdriver_service, options=chrome_options)
+
+    #spotipy object API
+    scope = "user-read-playback-state user-modify-playback-state user-read-currently-playing user-read-recently-played streaming user-library-read user-read-private user-library-modify user-read-playback-position playlist-read-private playlist-modify-private"
+    sp = spotipy.Spotify(auth_manager=SpotifyOAuth(client_id=cred.client_id, client_secret=cred.client_secret, redirect_uri=cred.redirect_uri, scope=scope))
+

```

Comment:
For these Variables you are creating, you should move these into the __init__(self,...) function and store them as self.VariableName

Reply:
Also, some of these variables should be Private from other classes. Any variable that can be used to modify the class should be kept as private. this is as simple as changing the variable name from "self.variable" to "self._variable" by adding an underscore to the front.